es710x/src/main/scala/tech/beshu/ror/es/handler/AclAwareRequestFilter.scala

```diff
@@ -162,6 +162,10 @@ class AclAwareRequestFilter(clusterService: RorClusterService,
         regularRequestHandler.handle(new GetAliasesEsRequestContext(request, esContext, aclContext, clusterService, threadPool))
       case request: IndicesAliasesRequest =>
         regularRequestHandler.handle(new IndicesAliasesEsRequestContext(request, esContext, aclContext, clusterService, threadPool))
+      // data streams
```

Comment:
why it cannot stay like it was before? I mean, the reflection-based request contexts are at the end of the matching code

Reply:
Some action requests matched the other case - `IndicesRequest.Replaceable`. It's fixed as we agreed